static/js/settings_users.js

```diff
@@ -188,8 +188,25 @@ function populate_users(realm_people_data) {
             },
             onupdate: reset_scrollbar($users_table),
         },
+        parent_container: $('#admin-user-list').expectOne(),
     }).init();
 
+    list.add_sort_function("role", function (a,b,c) {
+        if(page_params.is_admin) {
+            a == 1;
+        } else if (page_params.is_guest) {
+            a == 2;
+        } else {
+            c == 3;
+        }
```

Comment:
I think these should be admin=1, member=2, guest=3; that's more in line with what users expect.

Reply:
Ohh I'm sorry for that actually I asked her to create PR so that I that see if she's going right even though her commits weren't ready.